_data/projects/Mudlet.yml

```
@@ -0,0 +1,13 @@
+name: Mudlet
+desc: A cross-platform, open source, and super fast MUD client with scripting in Lua
+site: https://www.mudlet.org
+tags:
+  - mud
+  - text
+  - game
+  - c++
+  - lua
+  - multiplatform
+upforgrabs:
+  name: good first issue
+  link: https://github.com/Mudlet/Mudlet/issues?q=label%3A%22good+first+issue%22
```

:art: a simpler link here:

```suggestion
      link: https://github.com/Mudlet/Mudlet/labels/good%20first%20issue
```